mirrord/cli/src/connection.rs

```diff
@@ -129,9 +129,8 @@ where
             .await
             .map_err(CliError::KubernetesApiFailed)?;
 
-        let mut detect_openshift_task = progress.subtask("detecting OpenShift...");
-        if (k8s_api.detect_openshift(&mut detect_openshift_task).await).is_err() {
-            detect_openshift_task.warning("couldn't determine OpenShift");
+        if (k8s_api.detect_openshift(progress).await).is_err() {
+            progress.warning("couldn't determine OpenShift");
```

Comment:
We don't need this warning either + I think it makes sense to add logs in place of all the messages you remove

Reply:
Is "OpenShift was not detected." still going to show? If so please remove that as well